src/gui/windows/multiplayer.zig

```diff
@@ -56,17 +56,32 @@ fn join(_: usize) void {
 		ipAddress = "";
 	}
 	if(connection) |_connection| {
+		if (ipAddressEntry.currentString.items.len == 0) {
+			std.log.err("IP address cannot be empty", .{});
+			main.gui.windowlist.notification.raiseNotification("IP address cannot be empty");
+			return;
+		}
```

Comment:
Do we even still need this as a separate check? Isn't this mistake also captured by the check below?